Task/Taskfile.yml

```diff
@@ -3,8 +3,127 @@
 
 version: "3"
 
+set:
```

Comment:
This essentially runs `set -euo pipefail` which we always want.  Same with the other `Taskfile.yml`s